check.py

```diff
@@ -114,33 +119,30 @@ def get_branch():
     cleanup = os.getenv("INPUT_CLEANUP", "false").lower()
     file_types = os.getenv("INPUT_FILE_TYPES", "").split(",")
     print_all = os.getenv("INPUT_PRINT_ALL", "").lower()
-    white_listed_urls = os.getenv("INPUT_WHITE_LISTED_URLS", "").split(",")
-    white_listed_patterns = os.getenv("INPUT_WHITE_LISTED_PATTERNS", "").split(",")
+    white_listed_urls = urlproc.remove_empty(os.getenv("INPUT_WHITE_LISTED_URLS", "").split(","))
+    white_listed_patterns = urlproc.remove_empty(os.getenv("INPUT_WHITE_LISTED_PATTERNS", "").split(","))
+    white_listed_files = urlproc.remove_empty(os.getenv("INPUT_WHITE_LISTED_FILES", "").split(","))
     force_pass = os.getenv("INPUT_FORCE_PASS", "false").lower()
     retry_count = int(os.getenv("INPUT_RETRY_COUNT", 1))
     timeout = int(os.getenv("INPUT_TIMEOUT", 5)) # seconds
 
-    # Are whitelisted urls provided, or an empty string?
-    white_listed_urls = [x for x in white_listed_urls if x not in ["", None]]
-    white_listed_patterns = [x for x in white_listed_patterns if x not in ["", None]]
-
     # clone project repo if defined
     base_path = os.environ.get("GITHUB_WORKSPACE", os.getcwd())
 
     # Alert user about settings
-    print("  base path: %s" % base_path)
-    print("   git path: %s" % git_path)
-    print("  subfolder: %s" % subfolder)
-    print("     branch: %s" % branch)
-    print("    cleanup: %s" % cleanup)
-    print(" file types: %s" % file_types)
-    print("  print all: %s" % print_all)
-    print("  whistlist: %s" % white_listed_urls)
-    print("   patterns: %s" % white_listed_patterns)
-    print(" force pass: %s" % force_pass)
-    print("retry count: %s" % retry_count)
-    print("    timeout: %s" % timeout)
-
+    print("      base path: %s" % base_path)
+    print("       git path: %s" % git_path)
+    print("      subfolder: %s" % subfolder)
+    print("         branch: %s" % branch)
+    print("        cleanup: %s" % cleanup)
+    print("     file types: %s" % file_types)
+    print("      print all: %s" % print_all)
+    print(" url whitetlist: %s" % white_listed_urls)
+    print("   url patterns: %s" % white_listed_patterns)
+    print("  file patterns: %s" % white_listed_files)
```

Comment:
For the sake of consistency, I think this should be `files whitelist`.

Reply:
Also I have the same question concerning why only whitelisted files and not patterns too? I know we have the subfolder feature but still, I think there can be some cases, where it is practical to have a pattern.

Reply:
It would be hugely redundant, and I think the redundancy would be confusing. For example let's say I want to skip over README.md. Do I add that as a pattern, or as a file path? If I do it your way and require the path, I would need to do:

```
white_listed_files: "/github/workspace/README.md"
```
but with the current implementation, I can do:

```
white_listed_files: README.md
```
and it's going to meet my expectation and work. If you want to do the work to add it, I am okay with this, but I don't think that adding this feature improves usability for the reasons above.